formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -412,6 +415,287 @@
       },
     },
   },
+  uk_market_conformity_assessment_body_metadata: {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      uk_market_conformity_assessment_body_name: {
+        type: "string",
+      },
+      updated_at: {
+        type: "string",
+      },
+      uk_market_conformity_assessment_body_number: {
+        type: "string",
+      },
+      uk_market_conformity_assessment_body_type: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "approved-body",
+            "recognised-third-party-organisation",
+            "technical-assessment-body",
+            "user-inspectorate",
+          ],
+        },
+      },
+      uk_market_conformity_assessment_body_registered_office_location: {
+        type: "string",
+        enum: [
+          "united-kingdom",
+          "japan",
+          "switzerland",
+          "united-states-of-america",
+          "australia",
+          "new-zeeland",
+          "israel",
+          "canada",
+        ],
+      },
+      uk_market_conformity_assessment_body_testing_locations: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "north-macedonia",
```

Comment:
Same comment about confusing order.

Reply:
I've updated the order. Looks like it was alphabetical with UK at the top? originally? that's what I've gone with at least